Vagrantfile

```diff
@@ -185,6 +191,34 @@ def deb_common(config, name, extra: '')
   )
 end
 
+def deb_docker(config)
+  config.vm.provision 'install Docker using apt', run: 'always', type: 'shell', inline: <<-SHELL
```

Comment:
Is `run: 'always'` here on purpose? Do we intend to attempt to install docker on every `vagrant up/reload` operation? Details [here](https://www.vagrantup.com/docs/provisioning/basic_usage.html#run-once-always-or-never).